README.md

```diff
@@ -37,10 +37,7 @@ To submit your homework, zip up the contents of your git repo in a .zip file.
 Please remove the .git directory, and *anything else that might identify you*.
 If you want to include any comments with your submission, add them to the
 COMMENTS file. If there's anything important for us to know about how to run
-your code, please include that information in the COMMENTS file. We're quite familiar with Go,
-Ruby, Javascript, and Python, but if your submission is in some other
-language (which is OK), please let us know how we can set up our environment
-to run your code. Feel free to include any other information relevant to your
+your code, please include that information in the COMMENTS file. Feel free to include any other information relevant to your
```

Comment:
We should ask them to remember to stay anonymous. Some COMMENTS files say the person's name.